pyproject.toml

```
@@ -1,6 +1,71 @@
 [build-system]
-requires = ["setuptools"]
-build-backend = "setuptools.build_meta"
+requires = ["flit_core >=3.2,<4"]
+build-backend = "flit_core.buildapi"
+
+[project]
+name = "jupyter_enterprise_gateway"
+description = "A web server for spawning and communicating with remote Jupyter kernels"
+license = { file = "LICENSE.md" }
+keywords = ["Interactive","Interpreter","Kernel", "Web", "Cloud"]
+classifiers = [
+  "Intended Audience :: Developers", "Intended Audience :: Science/Research",
+  "Intended Audience :: System Administrators",
+  "License :: OSI Approved :: BSD License",
+  "Programming Language :: Python",
+  "Programming Language :: Python :: 3",
+  "Programming Language :: Python :: 3 :: Only",
+  "Programming Language :: Python :: 3.7",
+  "Programming Language :: Python :: 3.8",
+  "Programming Language :: Python :: 3.9",
+  "Programming Language :: Python :: 3.10"
+]
+requires-python = ">=3.7"
+dependencies = [
+  "docker>=3.5.0",
+  "future",
+  "jinja2>=3.1",
+  "jupyter_client~=6.1",
```

I'd like to start documenting "caps" when they are necessary (and since we're here...):
```suggestion
  "jupyter_client~=6.1",  # Remove cap once EG supports kernel provisioners
```